doc/news/_preparation_next_release.md

```diff
@@ -166,6 +167,19 @@ _(Michael Tucek)_
 
 - Really add all tools when using `-DTOOLS=ALL`. _(Markus Raab)_
 - ZeroMQ Hub: fix compilation and man page. _(Markus Raab)_
+
+### KDB
+
+- `kdb set`, `kdb meta-set`: Only allow writes to the cascading namespace if the lookup succeeds (Otherwise, the operation is ambiguous and therefore aborted).
+  No more guessing of namespaces in case a cascading key is given (`user:, system`: for `kdb set`, `spec:` for `kdb meta-set`), _(Alexander Firbas)_
+- `kdb set`, `kdb meta-set`: Validation of keys can no longer be bypassed by using non-cascading keys (except with the new --force (-f) option). _(Alexander Firbas)_
+- Disable `-N/--namespace` option in all kdb subcommands _(Alexander Firbas)_
+- Implement new name part getter commands `kdb namespace`, `kdb basename`, `kdb dirname` and `kdb dirkey`. _(Alexander Firbas)_
```

Comment:
might need update (dirkey not needed?) Please add other 3 names to doc/help/elektra-glossary.md

Reply:
The releasenotes got updated; I only added dirname as basename and namespace are already described in that file.